cmb2-conditionals.php

```diff
@@ -21,21 +21,36 @@ function cmb2_conditionals_load_actions()
 	define('CMB2_CONDITIONALS_PRIORITY', 99999);
 
 	add_action('admin_init', 'cmb2_conditionals_hook_data_to_save_filtering', CMB2_CONDITIONALS_PRIORITY);
-	add_action('admin_footer', 'cmb2_conditionals_footer', CMB2_CONDITIONALS_PRIORITY);
 }
 
+add_action('admin_enqueue_scripts', 'cmb2_conditionals_footer', CMB2_CONDITIONALS_PRIORITY);
 /**
  * Decides whether include the scripts or not.
  */
-function cmb2_conditionals_footer()
+function cmb2_conditionals_footer($hook)
 {
-	global $pagenow;
-
-    if(!in_array($pagenow, array('post-new.php', 'post.php'))) {
-    	return;
-    }
-
-	wp_enqueue_script('cmb2-conditionals', plugins_url('/cmb2-conditionals.js', __FILE__ ), array('jquery'), '1.0.2', true);
+  if( $hook != 'post.php' && $hook != 'post-new.php' ){
+    return;
+  }
+
+  $dir = trailingslashit( dirname( __FILE__ ) );
```

Comment:
Rather than reinvent the wheel, since this is a CMB2 extension, you could use the CMB2 utility method:
```php
$url = CMB2_Utils::get_url_from_dir( trailingslashit( dirname( __FILE__ ) ) )
```